### ISSUE TITLE
min-width doesn't recalc after resizing window to full screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#130]
```
What steps will reproduce the problem?
1. See attached html+css. 
2. Resize window to width < 600px. H. scrollbar appears. Content width is
600px, window width is < 600px
3. Resize window to fullscreen, window width must be more than 600px.

What is the expected output? What do you see instead?
1. Window width is now more than 600px, but content is still 600px width.
2. Content width must be equal to window width 


What version of the product are you using? On what operating system?
IE7.js beta3
Browser is IE6.0.3790.3959




```

Original issue reported on code.google.com by `olegelif...@gmail.com` on 19 Nov 2008 at 1:46

Attachments:
- [mw-test.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-130/comment-0/mw-test.html)
- [mw-test.css](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-130/comment-0/mw-test.css)

**Comment by GoogleCodeExporter** [comment]
```
This bug seems to apply only when min-width is used on the BODY element. 
Compare the
two examples attached.
```

Original comment by `svar...@gmail.com` on 23 Jul 2009 at 2:03

Attachments:
- [minwidth-body.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-130/comment-1/minwidth-body.html)
- [minwidth-div.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-130/comment-1/minwidth-div.html)

**Comment by GoogleCodeExporter** [comment]
```
It also seems to occur in block elements with overflow-y:auto.
```

Original comment by `svar...@gmail.com` on 28 Jul 2009 at 2:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 13 Feb 2010 at 1:47
- Changed state: **Fixed**
